sdk/src/test/java/co/omise/android/GooglePayTest.kt

```diff
@@ -0,0 +1,131 @@
+package co.omise.android
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import co.omise.android.request.GooglePay
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.json.JSONObject
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+
+
+@RunWith(AndroidJUnit4::class)
+class GooglePayTest {
```

Comment:
Usually, class and unit test class should have same package name. Could you make a change?

Reply:
changed here https://github.com/omise/omise-android/pull/213/commits/9b6b980e09601648c2a7b31592d736498f4cc471